pkg/deps/config/suppliers.go

```diff
@@ -540,7 +541,7 @@ func NewSupplyParamsCacheFn[T any](opts ...querycache.CacheOption[client.ParamsC
 			return depinject.Configs(deps, depinject.Supply(noopParamsCache)), nil
 		}
 
-		paramsCache, err := querycache.NewParamsCache[T](memory.WithTTL(0))
+		paramsCache, err := querycache.NewParamsCache[T](memory.WithTTL(math.MaxInt64))
```

Comment:
#PUC that when we switch to event-driven cache warming, that we should use a "real TTL" (i.e. a block + some wiggle room)